sites/svelte.dev/src/routes/(authed)/repl/[id].json/+server.js

```diff
@@ -43,7 +43,10 @@ export async function GET({ params }) {
 		const res = await fetch(`${API_BASE}/docs/svelte/examples/${params.id}`);
 
 		if (!res.ok) {
-			return new Response(await res.json(), { status: res.status });
+			return new Response(await res.json(), {
+				status: res.status,
+				headers: { 'Content-Type': 'application/json' }
+			});
```

Comment:
not sure i understand why this isn't `throw error(res.status, await res.json())` or even just `throw error(res.status)`?

Reply:
the second param to `error` must be an `App.Error`. I don't know exactly what `res.json()` would contain and if it would adhere to that type